src/App.jsx

```diff
@@ -3,69 +3,134 @@ import './App.css'
 import MovieList from './MovieList'
 import Header from './Header'
 import Footer from './Footer'
-
+import SideBar from './SideBar'
 
 const App = () => {
   const [loadedMovies, setLoadedMovies] = useState([]);
   const [page, setPage] = useState(1); //Page number of next page to load
-  const baseURL = "https://image.tmdb.org/t/p/w500";
   const [searchTerm, setSearchTerm] = useState('');
+  const [genre, setGenre] = useState('');
```

Comment:
Instead of providing an empty string for the default value, it would meake more sense to use `null` to represent no value